package.json

```diff
@@ -29,6 +29,7 @@
     "backstopjs:init": "backstop init --config=backstop.js",
     "travis:backstopjs": "backstop test --config=backstop-travis.js --docker",
     "screenshot": "storybook-chrome-screenshot -p 9001 -c .storybook",
+    "lint": "npm run lint:js && npm run lint:php && npm run lint:css",
```

Comment:
Oh, good point. For now I just added that as a convenience because it wasn't in use and thought it was nice—it's actually not used by other build steps directly, as the PHP linting takes nearly a minute to run and wasn't previously part of the build process.